### PR TITLE
[patch] Fix various typos.

### PR DESCRIPTION
**Pull request by dtzSiFive**

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -1402,7 +1402,7 @@ wire b: UInt
 printf(clk, cond, "a in hex: %x, b in decimal:%d.\n", a, b) : optional_name
 ```
 
-On each positive clock edge, when the condition signal is high, the printf
+On each positive clock edge, when the condition signal is high, the `printf`{.firrtl}
```
Comment (seldridge) [inline spec.md:1405]: Nit: 80-column. Likely other places, too.

**Review by seldridge — APPROVED** [review]
LGTM

80-column nits and use the `[patch] label`. This doesn't need to update the revision history as there's already a generic spelling/grammar issues line item.

File: spec.md (hunk 1 above)
Comment (dtzSiFive) [reply spec.md]: Sure thing, done!

**Comment by dtzSiFive** [comment]
> 80-column nits and use the `[patch] label`. This doesn't need to update the revision history as there's already a generic spelling/grammar issues line item.

Done, thanks for the guidance.

**Review by azidar — APPROVED** [review]

**Pull request merged**